aclk/aclk_lws_wss_client.c

```diff
@@ -0,0 +1,340 @@
+#include "aclk_lws_wss_client.h"
+
+#include "libnetdata/libnetdata.h"
+
+#define UNUSED(x) (void)(x)
+
+static int aclk_lws_wss_callback(struct lws *wsi, enum lws_callback_reasons reason, void *user, void *in, size_t len);
+
+struct aclk_lws_wss_perconnect_data {
+    int todo;
+};
+
+//TODO config??
+int aclk_lws_wss_use_ssl = 1;
+
+struct lws_wss_packet_buffer {
+	unsigned char* data;
+	size_t data_size;
+	struct lws_wss_packet_buffer *next;
+};
+
+static inline struct lws_wss_packet_buffer *lws_wss_packet_buffer_new(void* data, size_t size)
+{
+	struct lws_wss_packet_buffer *new = callocz(1, sizeof(struct lws_wss_packet_buffer));
+	if(data) {
+		new->data = mallocz(LWS_PRE+size);
+    	memcpy(new->data+LWS_PRE, data, size);
```

Comment:
There is a problem with tabulation here.

Reply:
fixed in #7988